### PR TITLE
Jt hab 41

### PR DESCRIPTION
**Pull request by jwinnfeild2517**
Survey Cards for Favorites Page
https://bostoncommonpress.atlassian.net/browse/HAB-37

### CODE REVIEW

File: src/components/Cards/SurveyCard/index.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  return El;
+};
+
+const SurveyCard = ({
+  title,
+  subTitle,
+  handleSubmit,
+  surveyType,
+}) => {
+  const [errorMesssge, setErrorMessage] = useState();
```
Comment (arindoneATK) [inline src/components/Cards/SurveyCard/index.js:110]: looks like you got a spelling error here! errorMessage

File: src/components/Cards/SurveyCard/index.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      setErrorMessage(null);
+    } else {
+      setErrorMessage('Please make a selection');
+    }
+  };
+
+  return (
+    <SurveyCardWrapper>
+      <Title data-testid="survey-title">{title}</Title>
+      {!successMessage ? (
```
Comment (arindoneATK) [inline src/components/Cards/SurveyCard/index.js:126]: I think all the ternaries here can be difficult to read at first glance...I'll add a snippet that may be cleaner for ya in a minute.

File: src/components/Cards/SurveyCard/data.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  'gluten free',
+  'vegetarian',
+];
+
+export const npsData = [
+  'Not At All',
+  'Not Likely',
+  'Neither',
+  'Likely',
+  'Verfy Likely',
```
Comment (s-bridges) [inline src/components/Cards/SurveyCard/data.js:17]: 'Very Likely'

File: src/components/Cards/SurveyCard/index.js (hunk 2 above)
Comment (arindoneATK) [reply src/components/Cards/SurveyCard/index.js:126]: `    <>
        <SubTitle data-testid="survey-subtitle">{subTitle}</SubTitle>
        <FormEl surveyType={surveyType} handleSubmit={handleResponseSubmit} />
    </>
}
{successMessage && <SuccessMessage>{successMessage}</SuccessMessage>}
{errorMesssge && <ErrorMessage>{errorMesssge}</ErrorMessage>}`

Comment (arindoneATK) [reply src/components/Cards/SurveyCard/index.js:126]:
```
{!successMessage && 
    <>
        <SubTitle data-testid="survey-subtitle">{subTitle}</SubTitle>
        <FormEl surveyType={surveyType} handleSubmit={handleResponseSubmit} />
    </>
}
{successMessage && <SuccessMessage>{successMessage}</SuccessMessage>}
{errorMesssge && <ErrorMessage>{errorMesssge}</ErrorMessage>}
```

**Review by arindoneATK — CHANGES_REQUESTED** [review]
Some spelling and some code clean up, but otherwise looks good!

File: src/components/Cards/SurveyCard/index.js (hunk 1 above)
Comment (jwinnfeild2517) [reply src/components/Cards/SurveyCard/index.js]: updated

File: src/components/Cards/SurveyCard/index.js (hunk 2 above)
Comment (jwinnfeild2517) [reply src/components/Cards/SurveyCard/index.js]: updated, thanks

File: src/components/Cards/SurveyCard/data.js (hunk 3 above)
Comment (jwinnfeild2517) [reply src/components/Cards/SurveyCard/data.js]: nice catch, thanks

**Review by s-bridges — APPROVED** [review]
Looks good! Nice job!

**Review by arindoneATK — APPROVED** [review]
Nice refactor! You still have a spelling mistake on errorMessage...you have errorMesssage now (with 3 s)

**Pull request merged**